mvpa2/algorithms/searchlight_hyperalignment.py

```diff
@@ -143,6 +147,13 @@ def _call(self, ds):
                         fs[seed_index] = max(fs)
                     features_selected.append(fselector(fs))
                 ds = [sd[:, fsel] for fsel, sd in zip(features_selected, ds)]
+        elif np.sum(nfeatures_bin) > 0:
```

Comment:
a bit faster could be `elif np.any(nfeatures_bin)` since it would stop right when it finds at least a single one... and it would be somewhat more semantically better described intent ;-)